cstar/tests/blueprints/fixtures.py

```diff
@@ -140,11 +140,11 @@ def modify_yaml(
 
 
 @pytest.fixture
-def test_blueprint_as_path(test_blueprint_as_dict, tmp_path) -> Callable[[str], Path]:
+def blueprint_to_path(blueprint_to_dict, tmp_path) -> Callable[[str], Path]:
     """Given the name of a pre-defined blueprint, returns it as a (temporary) path to an on-disk file."""
 
     def _blueprint_as_path(name: str, local: bool = False) -> Path:
-        blueprint_dict = test_blueprint_as_dict(name, local=local)
+        blueprint_dict = blueprint_to_dict(name, local=local)
```

Comment:
Renaming these to remove the `test_*` prefix is reasonable, but `blueprint_to_path` as opposed to `blueprint_as_path` I feel is a little misleading. When called in user (testing) code, this function doesn't accept a blueprint, it's used as an alternative way to way to fetch one, but `blueprint_to` to me implies that it should be passed a blueprint object to change.

Reply:
changed in [2a6ee87](https://github.com/CWorthy-ocean/C-Star/pull/108/commits/2a6ee87c427752851a19e99d94dc9551316cd34d)